build-system/tasks/bundle-size/APPROVERS.json

```diff
@@ -0,0 +1,18 @@
+{
+  "dist/v0.js": {
+    "approvers": ["ampproject/wg-performance", "ampproject/wg-runtime"],
+    "threshold": 0.1
+  },
+  "dist/amp4ads-v0.js": {
+    "approvers": ["ampproject/wg-ads"],
+    "threshold": 0.1
+  },
+  "dist/v0/amp-story-0.1.js": {
+    "approvers": ["ampproject/wg-stories"],
+    "threshold": 0.5
+  },
+  "dist/v0/amp-story-1.0.js": {
+    "approvers": ["ampproject/wg-stories"],
+    "threshold": 0.5
```

Comment:
Where is this file currently used? Do you think it's worth changing it to jsonc or json5 so it can support inline comments?

Reply:
I prefer using plain JSON so I won't have to pull in more dependencies in the bot. The README should provide enough context

Reply:
I would support json5 here